gcp-cloud-run/src/main.tf

```diff
@@ -1,19 +1,6 @@
-resource "google_cloud_run_service" "main" {
-  name     = var.md_metadata.name_prefix
-  location = local.gcp_region
-
-  template {
-    spec {
-      containers {
-        image = var.container_repository
-      }
-    }
-  }
-  metadata {
-    annotations = {
-      # For valid annotation values and descriptions, see
-      # https://cloud.google.com/sdk/gcloud/reference/run/deploy#--ingress
-      "run.googleapis.com/ingress" = "internal-and-cloud-load-balancing"
-    }
-  }
+module "application_cloud_run" {
+  source          = "github.com/massdriver-cloud/terraform-modules//gcp-application-cloud-run"
```

Comment:
If it’s specific to a Massdriver app let’s make sure we name these consistently. massdriver-application-